assets/scss/core/_variables.scss

```diff
@@ -83,3 +83,15 @@ $sides: (top, bottom, left, right); /// Directions
     }
   }
 }
+
+.text-align-r {
```

Comment:
How about using BEM here? `text-align--right` The same with the other classes, in this file.